lib/tree.rb

```diff
@@ -1,38 +1,60 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = []
+    @alive = 0
   end
 
   def age!
+    @age += 1
+    @height += 1
+    @alive = false if @age > 15
+    add_apples if @age % 3 == 0
   end
 
   def add_apples
+    (1..10).each do |idx|
```

Comment:
Could you also do something like this... 

```
10.times { @apples.push Apple.new("Red", rand(4..6)) }
```

I'd assume the diameter would have less of a variance :)

https://ruby-doc.org/core-2.2.2/Integer.html#method-i-times